src/index.html

```diff
@@ -1,13 +1,307 @@
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./styles/icons/favicon.svg"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion-->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+                class="top-bar__logo top-bar__logo-white"
+                src="styles/images/Logo-White.svg"
+                alt="MyBike-Logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <div class="top-bar__phone">
+                <!-- <a href="tel:+1-234-5555-55-55" class="icon icon--phone">+1 234 5555-55-55</a> -->
+                <a href="tel:+1-234-5555-55-55" class="icon icon--phone top-bar__call"></a>
+              </div>
+
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">Take the Streets</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              class="top-bar__logo top-bar__logo-black"
+              src="styles/images/Logo-Black.svg"
+              alt="MyBike-Logo">
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav_item">
+                <a class="nav__link" href="#">Home</a>
+              </li>
+              <li class="nav_item">
+                <a class="nav__link" href="#about-us">About us</a>
+              </li>
+              <li class="nav_item">
+                <a class="nav__link" href="#compare-bikes">Compare Bikes</a>
+              </li>
+              <li class="nav_item">
+                <a class="nav__link" href="#details">Details</a>
+              </li>
+              <li class="nav_item">
+                <a class="nav__link" href="#contacts">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+          <a href="tel:+1-234-5555-55-55" class="menu__phone-number">
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
          <a href="tel:+123455555555" class="menu__phone-number">
```